### PR TITLE
Observation formatted comment documentation update

### PR DESCRIPTION
**Pull request by BS062721**
DSTU2:
DSTU2_Comment_information

R4:
R4_comment_information

### CODE REVIEW

File: content/millennium/r4/diagnostic/observation.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied
 
 _Implementation Notes_
 
-* The `comments` field may have RTF or other formatted data rather than simple text.
+* The `comments` field may have RTF or other formatted data rather than simple text. This is an issue that will be resolved in a future correction. We are also investigating markdown support.
```
Comment (beichhor) [inline content/millennium/r4/diagnostic/observation.md]: I think this note may be still too vague.

Comment (BS062721) [reply content/millennium/r4/diagnostic/observation.md]: This was suggested by Dennis, I am not involved in that conversation Seada can give more information on this.

Comment (SA030800) [reply content/millennium/r4/diagnostic/observation.md]: yes this is the note from dennis

> For DSTU2, we already state there is RTF.  We’d update the existing comment “This is an issue that will be resolved in a future correction” to instead say it’ll be resolved in R4.  For R4, we’d echo the existing comment about RTF / formatted data and state that we’re investigating markdown support.

Comment (BS062721) [reply content/millennium/r4/diagnostic/observation.md]: Updated it as discussed: https://github.com/cerner/fhir.cerner.com/pull/352/commits/56249e811fbbf88e70affc672b4f98dae6a85ef6

New output:

Screen Shot 2020-03-04 at 12 07 38 PM

**Comment by ebaranowski** [comment]
Screen Shot 2020-04-01 at 10 23 26 AM
Screen Shot 2020-04-01 at 10 23 55 AM

Validated

**Pull request merged**